manifests/base/deployment.yaml

```diff
@@ -1,29 +1,50 @@
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: tf-job-operator
+  name: training-operator
+  namespace: system
+  labels:
+    control-plane: kubeflow-training-operator
 spec:
+  selector:
+    matchLabels:
+      control-plane: kubeflow-training-operator
   replicas: 1
   template:
     metadata:
       labels:
-        name: tf-job-operator
-      annotations:
-        sidecar.istio.io/inject: "false"
+        control-plane: kubeflow-training-operator
     spec:
+      #      securityContext:
+      #        runAsNonRoot: true
       containers:
-      - args:
-        - -monitoring-port=8443
-        env:
-        - name: MY_POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: MY_POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        image: public.ecr.aws/j1r0q0g6/training/tf-operator
-        name: tf-job-operator
-      serviceAccountName: tf-job-operator
+        - command:
+            - /manager
+          # disable leader-elect now
```

Comment:
Remove the lines that are no longer needed?

Reply:
@Jeffwan should we be removing it or can be handled through another tracker which enables leader election. where do i find details of why it was disabled?

Reply:
This will be needed for beta release.  @DeeperMind  please create an issue to track it.